gustaf/spline/create.py

```diff
@@ -0,0 +1,229 @@
+"""gustaf/spline/create.py
+
+Create operations for spline geometries
+"""
+
+
+import itertools
+import logging
+import numpy as np
+
+
+def extrude(spline, axis=None):
+    """
+    Extrudes Splines ype objects of any spline-type object
+
+    Parameters
+    ----------
+    spline : Gustaf-Spline
+    axis : Extrusion axis
+    """
+    from gustaf.spline.base import GustafSpline
+
+    # Check input type
+    from gustaf.spline.base import GustafSpline
+    if not issubclass(type(spline), GustafSpline):
+        raise NotImplementedError("Extrude only works for splines")
+
+    # Check axis
+    if axis is not None:
+        axis = np.asarray(axis)
+        if len(axis.shape) == 0:
+            axis = np.asarray([axis])
+    else:
+        raise ValueError("No extrusion Axis given")
+
+    # Check Axis dimension
+    if not (spline.control_points.shape[1] == axis.shape[0]):
+        raise ValueError(
+            "Dimension Mismatch between extrusion axis and spline."
+        )
+
+    # Start Extrusion
+    arguments = {}
```

Comment:
```suggestion
    dict_spline = dict()
```
we call this kind of things `dict_spline`  + explicit `dict()`